src/lib/components/particles/stacked-bar/index.jsx

```diff
@@ -16,22 +18,26 @@ export function StackedBar({ stack, width, height, createSVG = true }) {
     }
   }, [stack, width, height])
 
+  styles = mergeStyles({ ...defaultStyles }, styles)
+
   let totalWidth = 0
   const content = stack.map((d, index) => {
     const itemWidth = Math.round(d.fraction * width)
-    const textColor = isDarkColor(d.fill) ? 'fill-white' : 'fill-neutral-7'
+    const textColor = isDarkColor(d.fill) ? '#FFF' : '#121212'
```

Comment:
can of course be added in a separate PR but can we use CSS variables here? eg  `var(--primary-text)` etc

Reply:
Yes, definitely. I think this depends on generating fill styles for all colours in our colour scheme though. I'll pick that up later.